cypress/e2e/pages/articles/testsForLiteOnly.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/prefer-default-export */
+import { liteEnabledServices } from '#app/components/LiteSiteCta/liteSiteConfig';
+
+export const testsForLiteOnly = ({ service, pageType }) => {
+  describe(`Running testsForLiteOnly for ${service} ${pageType}`, () => {
+    describe('CTA: Lite', () => {
+      it('should render a call to action component', () => {
+        if (liteEnabledServices.includes(service)) {
+          cy.get('[data-e2e="lite-cta"]').should('be.visible');
+        }
+      });
+    });
+  });
+};
```

Comment:
When we get a link to the find out more article, it would be good to extend the e2e to check that once clicked they're brought to the page correctly

Reply:
Have done (I think!)